web/docs/tql2/operators.md

```diff
@@ -74,18 +82,14 @@ Operator | Description | Example
 [`load_tcp`](./operators/load_tcp.md) | Loads bytes from a TCP or TLS connection | `load_tcp "0.0.0.0:8090" { read_json }`
 [`load_udp`](./operators/load_udp.md) | Loads bytes from a UDP socket | `load_udp "0.0.0.0:8090"`
 [`load_zmq`](./operators/load_zmq.md) | Receives bytes from ZeroMQ messages | `load_zmq`
+[`diagnostics`](./operators/diagnostics.md) | Retrieves diagnostic events of managed pipelines | `diagnostics`
 [`metrics`](./operators/metrics.md) | Retrieves metrics events from a Tenzir node | `metrics "cpu"`
-[`subscribe`](./operators/subscribe.md) | Subscribes to events of a certain topic | `subscribe "topic"`
-
-<!--
-[`load`](./operators/load.md) | Load bytes according to a URL | `load "https://example.org/api/list"`
-[`from`](./operators/from.md) | | `from "/tmp/data.json"`
--->
 
 ## Output
 
 Operator | Description | Example
 :--------|:------------|:-------
+[`to`](./operators/to.md) | Write to an URI | `from "s3://examplebucket/obj.json.gz"` <!--at the top because its important-->
```

Comment:
If we say `from` "reads events from an URI", then this should probably be "writes events to an URI", right?